src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiration-notification/pwd-expiration-notification.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+
+import { AuthService } from '../../api/auth.service';
+import { CdPwdExpirationSettings } from '../../models/cd-pwd-expiration-settings';
+import { AuthStorageService } from '../../services/auth-storage.service';
+
+@Component({
+  selector: 'cd-pwd-expiration-notification',
+  templateUrl: './pwd-expiration-notification.component.html',
+  styleUrls: ['./pwd-expiration-notification.component.scss']
+})
+export class PwdExpirationNotificationComponent implements OnInit {
+  alertType: string;
+  expirationDays: number;
+  pwdExpirationSettings: CdPwdExpirationSettings;
+
+  constructor(private authService: AuthService, private authStorageService: AuthStorageService) {}
+
+  ngOnInit() {
+    this.authService.pwdExpirationSettings().subscribe((pwdExpirationSettings) => {
```

Comment:
I'm getting the following error:
`Property 'pwdExpirationSettings' does not exist on type 'AuthService'.`

Reply:
Yes, I know, I'm not finished yet.